src/diffusers/models/downsamplers.py

```diff
@@ -0,0 +1,318 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
(nit) would prefer the filename `downsampling.py` here so that people 100% understand that we're not talking about "samplers"

Reply:
100 percent agreed here. Done in f200ce4c.